### DISCUSSION TITLE
springboot/initializr

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#198] [Announcements]
# springboot/initializr

关于 Spring Initializr Spring 官方提供了 Spring Initializr 的方式来创建 Spring Boot 项目。网址如下： https://start.spring.io/ 打开后的界面如下：

https://tobebetterjavaer.com/springboot/initializr.html

**Comment by czxk579** [comment]
你好，你的springboot 教程写的很详细，给你点赞，我在学习的时候，关于热部署有个问题，在 pom.xml 文件中添加依赖如下：
<dependency>
    <groupId>org.springframework.boot</groupId>
    <artifactId>spring-boot-devtools</artifactId>
</dependency>
然后试着修改接口的返回值，发现没有按照预期的那样重载，新的修改没有生效，请问该教程中，关于热部署是不是还有额外的操作没有说明呢？

**Reply by huangyingjie** [reply]
高级设置里面还要设置，即使服务启动状态也重新编译
